### DISCUSSION TITLE
Included page and frontmatter metadata

### DISCUSSION THREAD

**Discussion by postb99** [McShelby/hugo-theme-relearn#698] [Q&A] [answered]
Hello,

Is it sure that an included page should not have frontmatter metadata?

My home page looks like:

```
{{% notice title="Cinq derniers poèmes ajoutés" style="note" color="" icon="" %}}
{{% include "last_poems.md" true %}}
{{% /notice %}}
```

If I add the metadata to my included page, it gets treated as text, so I see "+++ title = ..." in html markup.

I see that your included page CHANGELOG.md doesn't contain frontmatter metadata but rather a top-level heading.

I had to guess and try it after reading https://mcshelby.github.io/hugo-theme-relearn/shortcodes/include/ but prefer to understand fully why and why not...

Thanks

**Comment by McShelby** [comment, marked as answer]
Thats correct. An included page must not contain frontmatter. It is purely interpreted as Markdown.
